docker/bootstrap/Dockerfile.percona57

```diff
@@ -11,7 +11,7 @@ RUN apt-key adv --keyserver ha.pool.sks-keyservers.net \
     } | debconf-set-selections && \
     apt-get update && \
     apt-get install -y --no-install-recommends \
-        percona-server-server-5.7 libperconaserverclient18.1-dev && \
+        percona-server-server-5.7=5.7.14* libperconaserverclient18.1-dev && \
```

Comment:
This was actually done as a cache busting mechanism.

I noticed on our local builds when following the instructions we were just using the cached versions of percona, which were old.

I added this so that we could pin to the latest stable and force an update when it changes.

I agree that no pinning should be necessary, but I'm not entirely sure how to get around the caching issues more effectively.

Additional workflow advise would be most helpful here.